pkg/controllers/netpol/policy.go

```diff
@@ -476,6 +476,14 @@ func (npc *NetworkPolicyController) appendRuleToPolicyChain(policyChainName, com
 	args = append(args, "-m", "mark", "--mark", "0x10000/0x10000", "-j", "RETURN", "\n")
 	npc.filterTableRules[ipFamily].WriteString(strings.Join(args, " "))
 
+	// Extra logs to get more information about the policy dropping the packet via ulog2
+	logRuleComment := "\"rule to log dropped traffic\""
+	logComment := "\"DROP by policy: " + policyName + " in namespace: " + policyNamespace + "\""
+	logArgs := []string{"-A", policyChainName, "-m", "comment", "--comment", logRuleComment, "-m", "limit",
+		"--limit", "10/minute", "--limit-burst", "10", "-m", "mark", "!", "--mark", "0x10000/0x10000",
```

Comment:
Might be interesting to make the limit configurable, perhaps via an annotation on the policy, or a configuration option on the controller to set the global default? If I'm troubleshooting something and can't see the logs from it because the policy has already exceeded its logging limit for that window I might be kind of frustrated.
```suggestion
		"--limit", "10/minute", "--limit-burst", "10", "-m", "mark", "!", "--mark", "0x10000/0x10000",
```

Reply:
Good point. I was just copy/pasting the other NFLOG action. I also need to investigate @rbrtbnfgl's idea about adding this line only if we are in "debug" mode

Reply:
done!